charts/oes/config/oes-gate/gate.yml

```diff
@@ -286,6 +286,15 @@ allowUnauthenticatedAccess:
   agentAPI: {{ .Values.gate.config.agentAPI }}
   webhooks: {{ .Values.gate.config.webhooks }}
 
+message-broker:
```

Comment:
Do  not hardcode the values

Reply:
@saitejaopsmx Updated the gate configuration to fetch RabbitMQ username, pwd and port values from values.yaml file.